scripts/sdk/build.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+# Exit whenever one of the commands fail with a non-zero exit code
+set -e
+set -o pipefail
```

Comment:
As the comment describes: `Exit whenever one of the commands fail with a non-zero exit code`